tests/functional/adapter/test_aliases.py

```diff
@@ -43,6 +74,31 @@ def project_config_update(self):
                 },
             },
         }
+
     @pytest.fixture(scope="class")
     def macros(self):
-        return {"bigquery_cast.sql": MACROS__BIGQUERY_CAST_SQL, "expect_value.sql": MACROS__EXPECT_VALUE_SQL}
+        return {
+            "bigquery_cast.sql": MACROS__BIGQUERY_CAST_SQL,
+            "expect_value.sql": MACROS__EXPECT_VALUE_SQL
+        }
+
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "schema.yml": MODELS_SCHEMA_YML,
+            "model_a.sql": MODELS_DUPE_CUSTOM_DATABASE_A,
+            "model_b.sql": MODELS_DUPE_CUSTOM_DATABASE_B
+        }
+
+    @pytest.fixture(scope="function")
```

Comment:
We want to do this clean up after each test?

Reply:
yeah just like with `override_database` bigquery needs the extra call for cleanup due to change in shcema not being seen, tried making it a class scope call that didn't work this seems to